components/Common/CodeBox/CodeTabs/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { ArrowUpRightIcon } from '@heroicons/react/24/solid';
+import * as TabsPrimitive from '@radix-ui/react-tabs';
+import type {
+  ComponentProps,
+  FC,
+  PropsWithChildren,
+  ReactElement,
+} from 'react';
+import { useMemo, useState } from 'react';
+
+import Tabs from '@/components/Common/Tabs';
+import LocalizedLink from '@/components/LocalizedLink';
+
+import styles from './index.module.css';
+
+type CodeTabsProps = Pick<
+  ComponentProps<typeof Tabs>,
+  'tabs' | 'onValueChange' | 'defaultValue'
+> & {
+  linkUrl?: string;
+  linkText?: string;
+};
+
+export const CodeTabs: FC<PropsWithChildren<CodeTabsProps>> = ({
+  children,
+  linkUrl,
+  linkText,
+  ...props
+}) => (
+  <Tabs
+    {...props}
+    className={styles.root}
+    headerClassName={styles.header}
+    addons={
+      linkUrl !== undefined && linkText !== undefined ? (
```

Comment:
```suggestion
      linkUrl && linkText && (
```

No need to check for undefined and there's no need for a ternary. As if any of these fail, they will be undefined and pass undefined.